src/styles/BaseStyles/src/globals.module.css

```diff
@@ -0,0 +1,23 @@
+:root {
+  /*
+   * Base tokens
+   */
+  --denhaag-fontsize-xs: 0.625rem;
+  --denhaag-fontsize-s: 0.75rem;
+  --denhaag-fontsize-base: 1rem;
+  --denhaag-fontsize-lg: 1.25rem;
+  --denhaag-fontsize-xl: 1.5rem;
+  --denhaag-fontsize-2xl: 2rem;
+  --denhaag-fontsize-3xl: 3rem;
```

Comment:
It's a bit unclear to me whether these should be brand tokens or design system tokens. In my opinion, the font sizes are not very brand specific, and having a fontsize scale defined in the basestyles will make it easier for developers to easily change the font sizes in the component library, without needing to change all component specific fontsize tokens.